cmd/storage.go

```diff
@@ -180,50 +180,23 @@ var storageListCmd = &cobra.Command{
 		applicationName, err := application.GetCurrent(client)
 		checkError(err, "")
 		projectName := project.GetCurrent(client)
-		componentName := getComponent(client, storageComponent, applicationName, projectName)
-
-		if componentName == "" {
-			fmt.Println("No component selected")
-			os.Exit(1)
-		}
 
-		storageList, err := storage.List(client, componentName, applicationName)
-		checkError(err, "")
-
-		hasMounted := false
-		hasUnmounted := false
-		tabWriterMounted := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-		tabWriterUnmounted := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-
-		//create headers
-		fmt.Fprintln(tabWriterMounted, "NAME", "\t", "SIZE", "\t", "PATH")
-		fmt.Fprintln(tabWriterUnmounted, "NAME", "\t", "SIZE")
-
-		for _, storage := range storageList {
-			if storage.Path != "" {
-				if !hasMounted {
-					hasMounted = true
-				}
-				fmt.Fprintln(tabWriterMounted, storage.Name, "\t", storage.Size, "\t", storage.Path)
-			} else {
-				if !hasUnmounted {
-					hasUnmounted = true
-				}
-				fmt.Fprintln(tabWriterUnmounted, storage.Name, "\t", storage.Size)
+		if storageAllListflag {
+			if storageComponent != "" {
+				fmt.Println("Invalid arguments. component name is not needed")
+				os.Exit(1)
+			}
+			componentList := getComponentList(client, applicationName, projectName)
```

Comment:
I guess we can directly use component.List here instead of creating a new function in cmdutils